cpp/src/backends/torch_scripted/handler/base_handler.hh

```diff
@@ -70,12 +74,30 @@ class BaseHandler {
       std::shared_ptr<torchserve::InferenceRequestBatch>& request_batch,
       std::shared_ptr<torchserve::InferenceResponseBatch>& response_batch) {
     try {
+      auto start_time = std::chrono::high_resolution_clock::now();
+
       std::map<uint8_t, std::string> idx_to_req_id;
       auto inputs =
           Preprocess(device, idx_to_req_id, request_batch, response_batch);
       auto outputs =
           Inference(model, inputs, device, idx_to_req_id, response_batch);
       Postprocess(outputs, idx_to_req_id, response_batch);
+
+      auto stop_time = std::chrono::high_resolution_clock::now();
+      std::chrono::duration<double, std::milli> duration =
+          stop_time - start_time;
+      try {
+        auto& handler_time_metric =
+            torchserve::MetricsRegistry::GetMetricsCacheInstance()->GetMetric(
+                torchserve::MetricType::GAUGE, "HandlerTime");
+        handler_time_metric.AddOrUpdate(
+            std::vector<std::string>{manifest_->GetModel().model_name, "Model"},
+            duration.count());
+      } catch (std::runtime_error& e) {
+        TS_LOG(ERROR, e.what());
+      } catch (std::invalid_argument& e) {
+        TS_LOGF(ERROR, "Failed to record metric. {}", e.what());
+      }
```

Comment:
the request_id is not included in the log metrics. it cause different format model_metrics.log b/w python and cpp. it may break cx parser pipeline.